docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -1,23 +1,155 @@
+For the supervised training we use the following algorithms components:
+
+.. _semantic_segmentation_supervised_pipeline:
+
+- ``Augmentations``: Besides basic augmentations like random flip, random rotate and random crop, we use mixing images technique with different `photometric distortions <https://mmsegmentation.readthedocs.io/en/latest/api.html#mmseg.datasets.pipelines.PhotoMetricDistortion>`_.
+
+- ``Optimizer``: We use `Adam <https://arxiv.org/abs/1412.6980>`_ optimizer with weight decay set to zero and gradient clipping with maximum quadratic norm equals to 40.
+
+- ``Learning rate schedule``: For scheduling training process we use **ReduceLROnPlataeu** with linear learning rate warmup for 100 iterations. This method monitors a target metric (in our case we use metric on the validation set) and if no improvement is seen for a 'patience' number of epochs, the learning rate is reduced.
+
+- ``Loss function``: We use standart `Cross Entropy Loss <https://en.wikipedia.org/wiki/Cross_entropy>`_  to train a model.
+
+- Additionally, we use the **early stopping** to add adaptability to the training pipeline and prevent overfitting.
+
 **************
 Dataset Format
 **************
 
-To be added soon
+For the dataset handling inside OTX we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_.
+
+At this end we support `ADE20K <https://openvinotoolkit.github.io/datumaro/docs/formats/ade20k2020/>`_, `Cityscapes <https://openvinotoolkit.github.io/datumaro/docs/formats/cityscapes/>`__, `Pascal VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ and `Common Semantic Segmentation <https://openvinotoolkit.github.io/datumaro/docs/formats/common_semantic_segmentation/>`_ data formats.
+If you organized supported dataset format, starting training will be very simple. We just need to pass a path to the root folder and desired model template to start training:
+
+.. code-block::
+
+    $ otx {train, optimize} <model_template> --train-data-root <path_to_data_root>
+
+
+.. note::
+
+    OTX will recognize train and validation splits automatically. If you want to emphasize the validation dataset root path, you can simply add the ``--val-data-root <path_to_val_data_root>`` option.
+
+    Please, refer to our :doc:`dedicated tutorial <../../../tutorials/base/how_to_train/semantic_segmentation>` how to train, validate and optimize semantic segmentation model for more details.
+
+
+For now, we also support a dataset in the custom format below, where image-annotation pairs have the same name:
+
+::
+
+    data
+        ├── images
+            ├── train
+                ├── 0.jpg
+                ├── 1.jpg
+                ...
+                └── N.jpg
+            ├── val
+                ├── 0.jpg
+                ├── 1.jpg
+                ...
+                └── N.jpg
+        └── annotations
```

Comment:
This should be "masks" for datumaro format. Isn't it? @sungmanc

Reply:
We can pass `--train-data-root` to train and `--val-data-root` to val and the training will be started